src/magicbox/MagicBoxClear.ts

```diff
@@ -22,5 +22,12 @@ export class MagicBoxClear {
       .withLabel(l('Clear'))
       .withSelectAction(() => magicBox.clear())
       .build();
+
+    this.toggleTabindex(false);
+  }
+
+  public toggleTabindex(enabled: boolean) {
```

Comment:
I think it would be better to rename `enabled` to `hasText` or something similar that explains what the condition is.

Even better would be if we could access the `text` variable from inside the function instead of having to pass it in, but I don't know if that is easy?

Reply:
Not a fan of the second proposition, since we are in the MagicBoxClear button I don't want to go access the input's value, furthernore, we only call this method once from outside (on the setText)
But I can rename `enabled` to `hasText`